dist/aws/handler.py

```diff
@@ -59,6 +59,7 @@ def handler(event, context, get_deps=True, ret_pipeline=False):
     from podpac.core.pipeline import Pipeline
     from podpac.core.coordinates import Coordinates
     from podpac.core.utils import JSONEncoder, _get_query_params_from_url
+    import podpac.datalib
```

Comment:
I don't see this import being used, what does this do?

Reply:
datalib doesn't get imported by default when you import podpac (speed requirement). 

So, when a pipeline needs podpac.datalib.smap, it throws an error saying that datalib doesn't exist. 

This import takes care of that particular error. Should probably add a comment to that effect.

Reply:
👍